clang/lib/DPCT/DPCT.cpp

```diff
@@ -740,8 +734,32 @@ int runDPCT(int argc, const char **argv) {
   CudaPath = getCudaInstallPath(OriginalArgc, argv);
   DpctDiags() << "Cuda Include Path found: " << CudaPath << "\n";
 
-  RefactoringTool Tool(OptParser->getCompilations(),
-                       OptParser->getSourcePathList());
+  ArrayRef<std::string> SourcePathList;
```

Comment:
Let's use `SmallVector<std::string, 32>`, then we can avoid to use a `static std::vecto`r in L756